tutorials/advanced_features/Execution on an Emulator.ipynb

```diff
@@ -0,0 +1,633 @@
+{
```

Comment:
<p>nit: The base class is actually <code>RemoteConnection</code> but I don't think we want to go into those details </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pasqal-io/Pulser/pull/952/discussion/#comment-2526941758'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>